examples/org-viewer-react/webpack.config.js

```diff
@@ -0,0 +1,44 @@
+const path = require('path')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer')
+  .BundleAnalyzerPlugin
```

Comment:
I just have to say this is such a weird syntax for `require` haha. I guess prettier did this?